src/graphiql-explain/Explain.jsx

```diff
@@ -77,19 +51,34 @@ export function Content() {
   ])
 
   return (
-    <div style={{ height: '100%' }}>
+    <div className={styles.height100}>
       <input type="text" onChange={searchByPath} placeholder="Search by path" />
-      <table style={{ width: '100%' }}>
+      <table className={styles.width100}>
```

Comment:
I prefer to have a naming not strictly related to the style (eg. `container`, 'explain-data')